Examples/ImageGridManipulation/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+
+if ( NOT BUILD_TESTING )
+  return ()
+endif ()
+
+if ( NOT SimpleITK_TESTING_NOSHOW )
```

Comment:
No need for the conditional now that you removed all of the Show commands from the code.